Packs/CommonPlaybooks/ReleaseNotes/2_3_68.md

```diff
@@ -0,0 +1,9 @@
+
+#### Playbooks
+
+##### Detonate File - Generic
+
+- Updated the playbook to use the OPSWAT-Filescan *opswat-filescan-scan-file* command instead of the depreciated SNDBOX sub-playbook.
```

Comment:
```suggestion
- Updated the playbook to use the OPSWAT-Filescan *opswat-filescan-scan-file* command instead of the deprecated SNDBOX sub-playbook.
```